src/md/src/GettingStarted/ConfigurationOfReportPortalServer.md

```diff
@@ -60,8 +60,9 @@ Example of email server configuration for Yandex email server (detailed info cou
 
 ### Authorization configuration
 Admin of Report Portal instance can provide an access to external user. 
-As for now, Report Portal allows GitHub users to login via GitHub auth.
+As for now, Report Portal allows GitHub users to login via GitHub auth, Active Directory Authorization or LDAP Authorization.
```

Comment:
As for now, Report Portal allows GitHub users to login via GitHub OAuth2, Active Directory or LDAP